src/Provider/Doctrine/FieldDef.php

```diff
@@ -78,4 +78,35 @@ public static function reference($name)
             return $factory->get($name);
         };
     }
+
+    /**
+     * Defines a field to `get()` a collection of named entities from the factory.
+     *
+     * The normal semantics of `get()` apply.
+     * 
+     * Normally this means that the field gets a fresh instance of the named
+     * entity. If a singleton has been defined, a collection with a single instance will be returned.
+     *
+     * @param string $name
+     * @param int $numberOfInstances
+     *
+     * @throws \InvalidArgumentException
+     * @return callable
+     */
+    public static function references($name, $numberOfInstances = 3)
```

Comment:
To keep behaviour consistent, the default value for ``$numberOfInstances`` should be 1. See ``FixtureFactory::getList``